src/tools/webdev.md

```diff
@@ -4,68 +4,76 @@ description: Command-line tools for Dart web development.
 ---
 <!--?code-excerpt path-base="examples/ng/doc"?-->
 
-This page explains how to use `webdev` and
-a tool it depends on—`build_runner`—to 
-build, serve, and test your web apps.
-The [webdev][] package provides `webdev`,
-which wraps around the more general-purpose
-[`build_runner` tool.][build_runner]
-
-Usually you can use `webdev` instead of directly using `build_runner`.
-The only time most web app developers run `build_runner` is for tests.
+This page explains how to use `webdev` to compile your app and
+`build_runner` to test your app.
 
 ## Setup
 
-The easiest way to get `webdev` is to globally install it,
-so that it can be [in your PATH.][PATH]
-Before you can use `webdev`,
-your web app must depend on the
-build_runner and build_web_compilers packages.
+Install `webdev` for all users. This adds the tool [to your PATH][PATH]. 
+
+Before you can use `webdev`, Add dependencies to the [build_runner][] 
+and [build_web_compilers][] packages to your app. The `build_runner`
+package adds scripting capabilities to `webdev`.
 
+```terminal
+$ dart pub add build_runner build_web_compilers --dev
+```
 
 ### Installing and updating webdev
 
-[Globally install][] `webdev` using pub:
+Use `dart pub` to install `webdev` for [all users][].
 
 ```terminal
 $ dart pub global activate webdev
 ```
 
-Use the same command to update `webdev`.
-We recommend updating `webdev` whenever you update your Dart SDK
-or when `webdev` commands unexpectedly fail.
+Use the same command to update `webdev`. Update `webdev` when you
+update your Dart SDK or when `webdev` commands fail unexpectedly.
 
-[Globally install]: /tools/pub/cmd/pub-global
+[all users]: /tools/pub/cmd/pub-global
 
 
 ### Depending on build_* packages
 
-To use `webdev` or (in a web app context) `build_runner`,
-you must be in the root directory of a package that depends on 
-the **build_runner** and **build_web_compilers** packages.
-If you're testing the app,
-it must also depend on **build_test**.
+To use `webdev`, you must be in the root directory of a package that
+depends on the **build_runner** and **build_web_compilers** packages.
+If you're testing the app, it must also depend on **build_test**.
 
-To depend on these packages,
-add the following [dev dependencies][]
-to your app's `pubspec.yaml` file:
+To depend on these packages, add the following [dev dependencies][] to
+your app's `pubspec.yaml` file:
 
 <!--?code-excerpt "quickstart/pubspec.yaml (build dependencies)"?-->
-```
+```yaml
   dev_dependencies:
-    # ···
+    ···
```

Comment:
The hash is a YAML comment, I'm not sure if we validate the yaml snippets or not, but if we ever do, we should avoid causing trouble:

```suggestion
    # ···
```

Reply:
Yeah I suppose we added the hash in-case the user wanted to copy this snippet directly. 

Perhaps we should switch these instructions to `dart pub add` instead anyway though as those mentioned versions get out of date.